libs/community/langchain_community/vectorstores/kinetica.py

```diff
@@ -221,6 +221,7 @@ def __from(
         metadatas: Optional[List[dict]] = None,
         ids: Optional[List[str]] = None,
         collection_name: str = _LANGCHAIN_DEFAULT_COLLECTION_NAME,
+        schema_name: str = _LANGCHAIN_DEFAULT_SCHEMA_NAME,
```

Comment:
This is a breaking change because users may technically be invoking this by position

could you add place it as a last parameter and right before it use a `*` operator

So

```python
logger: ...,
*,
schema_name: str = ...,
**kwargs

```

Reply:
@eyurtsev Thanks for pointing this, I have modified the code to make this keyword only.